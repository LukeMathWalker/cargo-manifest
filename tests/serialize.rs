use cargo_manifest::{Manifest, Package};

#[test]
fn basic() {
    let manifest: Manifest<(), ()> = Manifest {
        package: Some(Package::new("foo".into(), "1.0.0".into())),
        ..Default::default()
    };

    let serialized = toml::to_string(&manifest).unwrap();
    insta::assert_snapshot!(serialized);
}
