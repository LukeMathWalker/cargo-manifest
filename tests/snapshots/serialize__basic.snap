---
source: tests/serialize.rs
expression: serialized
---
[package]
name = "foo"
version = "1.0.0"

